packages/client/README.md

```diff
@@ -205,26 +205,29 @@ const client = OpenFeature.getClient();
 client.setLogger(logger);
 ```
 
-### Named clients
+### Domains
 
-Clients can be given a name.
-A name is a logical identifier that can be used to associate clients with a particular provider.
-If a name has no associated provider, the global provider is used.
+Clients can be assigned to a domain.
+A domain is a logical identifier which can be used to associate clients with a particular provider.
+If a domain has no associated provider, the global provider is used.
 
 ```ts
-import { OpenFeature } from "@openfeature/web-sdk";
+import { OpenFeature, InMemoryProvider } from "@openfeature/web-sdk";
 
 // Registering the default provider
-OpenFeature.setProvider(NewLocalProvider());
-// Registering a named provider
-OpenFeature.setProvider("clientForCache", new NewCachedProvider());
+OpenFeature.setProvider(InMemoryProvider(myFlags));
+// Registering a provider to a domain
+OpenFeature.setProvider("in-memory", new InMemoryProvider(someOtherFlags));
```

Comment:
The domain in this example suggests using the provider name. I would try to lead users away from this practice towards naming scopes. See https://github.com/open-feature/spec/pull/229#issuecomment-1891083384

Reply:
I changed it but kept the term generic.